inception-ui-curation/src/main/java/de/tudarmstadt/ukp/clarin/webanno/ui/curation/component/model/AnnotatorSegment.java

```diff
@@ -79,28 +76,7 @@ public void setAnnotatorState(AnnotatorState aState)
         state = aState;
     }
 
-    public AnnotationSelection getAnnotationSelection()
-    {
-        return annotationSelection;
-    }
-
-    public void setAnnotationSelection(AnnotationSelection aAnnotationSelection)
-    {
-        annotationSelection = aAnnotationSelection;
-    }
-
-    public Map<String, Map<Integer, AnnotationSelection>> getSelectionByUsernameAndAddress()
-    {
-        return selectionByUsernameAndAddress;
-    }
-
-    public void setSelectionByUsernameAndAddress(
-            Map<String, Map<Integer, AnnotationSelection>> aSelectionByUsernameAndAddress)
-    {
-        selectionByUsernameAndAddress = aSelectionByUsernameAndAddress;
-    }
-
-    public boolean equals(UserAnnotationSegment segment)
+    public boolean equals(AnnotatorSegment segment)
```

Comment:
@Override ?

Reply:
I'm not sure if we even keep this class... also no idea what the "equals" does here - it does not override `equals(object)` - and if it did, we'd also have to override `hashCode`.